djconnectwise/sync.py

```diff
@@ -1633,6 +1633,40 @@ def _assign_field_data(self, instance, json_data):
                 '- skipping.'.format(instance.id, member_id)
             )
 
+        # Handle status- there are cases where no status field is provided,
```

Comment:
I don't believe making status or where requirements to sync is part of this issue. We allow creating without them so we should be able to sync without them being populated too.

Reply:
Got it, I have updated the synchronizer accordingly.